assets/javascripts/admintable.js

```diff
@@ -1,3 +1,18 @@
+var getUrlParameter = function getUrlParameter(sParam) {
```

Comment:
Don't duplicate the function `parseQueryParams()` which we already have in `openqa.js`.

Reply:
fixed.